pkg/modules/resiliency.go

```diff
@@ -0,0 +1,252 @@
+//  Copyright © 2023 Dell Inc. or its subsidiaries. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//       http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package modules
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	csmv1 "github.com/dell/csm-operator/api/v1"
+	drivers "github.com/dell/csm-operator/pkg/drivers"
+	"github.com/dell/csm-operator/pkg/logger"
+	utils "github.com/dell/csm-operator/pkg/utils"
+	rbacv1 "k8s.io/api/rbac/v1"
+	applyv1 "k8s.io/client-go/applyconfigurations/apps/v1"
+	acorev1 "k8s.io/client-go/applyconfigurations/core/v1"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	// DefaultPodmonArrayConnectivityPollRate -
+	DefaultPodmonArrayConnectivityPollRate = "60"
+	// DefaultPodmonAPIPort -
+	DefaultPodmonAPIPort = "8083"
+)
+
+var (
```

Comment:
These ENV vars are used inside Driver containers. In case if the below screenshot doesn't answer your question, please let me know.
![image](https://github.com/dell/csm-operator/assets/109620911/0f8bca0c-6701-44d3-8d40-1c6fff112c3d)